backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -992,6 +997,86 @@ databaseChangeLog:
                   name: default_device_type
               - column:
                   name: ordering_provider_id
+      rollback:
+        - addColumn:
+            tableName: organization
+            columns:
+              - column:
+                  name: clia_number
+                  type: text # irony
+                  remarks: The CLIA number for the facility that is ordering (and presumably also conducting) the tests.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: default_device_type
+                  remarks: The default device type (if any) for tests at this facility.
+                  type: uuid
+                  constraints:
+                    foreignKeyName: fk__organization__default_device_type
+                    referencedTableName: device_type
+              - column:
+                  name: ordering_provider_id
+                  type: uuid
+                  remarks: at what point does this blow up on me?
```

Comment:
Not sure if this is a leftover debugging comment?

Reply:
This is actually just copy/pasted from where the columns were originally added, since I just wanted to repeat the original instructions. Agreed on removing them though.